src/ape/pytest/fixtures.py

```diff
@@ -76,6 +77,13 @@ def project(self) -> ProjectManager:
 
         return self.project_manager
 
+    @pytest.fixture(scope="session")
+    def on_chain_contract(ContractInstance):
+        """
+        Access contract types and dependencies.
+        """
+        return ContractInstance.address()
```

Comment:
i dont think there is a class method `address` for ContractInstance, but if there was, that is not what we want here.

Return `self.chain_manager.instance_at` the method, the return type is a callable that takes an address and returns a `ContractInstance`